aio/aio-proxy/aio_proxy/search/filters.py

```diff
@@ -9,5 +13,28 @@ def filter_search(search, filters_to_ignore: list, **params):
     # params is the list of parameters (filters) provided in the request
     for param_name, param_value in params.items():
         if param_value is not None and param_name not in filters_to_ignore:
-            search = search.filter("term", **{param_name: param_value})
+            search = search.filter("term", **{get_es_field(param_name): param_value})
+    return search
+
+
+def filter_search_is_exist(search, filters_to_process: list, **params):
```

Comment:
The purpose of this function is not clear to me at all. I think a docstring should be added to explain what it does.